hoomd/ForceCompute.h

```diff
@@ -75,6 +75,42 @@ class PYBIND11_EXPORT ForceCompute : public Compute
     //! Computes the forces
     virtual void compute(uint64_t timestep);
 
+    //! Computes the parameters for meshes
+    virtual void precomputeParameter() { };
```

Comment:
Why are these methods in the base class `ForceCompute`? I think they should be defined in a subclass. If one doesn't exist, you could add a class that can be the parent of all mesh force computes.